ray-operator/controllers/ray/utils/util.go

```diff
@@ -32,6 +31,8 @@ const (
 	DefaultDomainName   = "cluster.local"
 )
 
+var log = logf.Log.WithName("controllers").WithName("RayCluster")
```

Comment:
It seems to be possible for RayJob/RayService controllers to call utility functions.

Reply:
It will be better to use `log := ctrl.LoggerFrom(ctx)` mentioned by @astefanutti.